src/display/editor/editor.js

```diff
@@ -1140,40 +1142,61 @@ class AnnotationEditor {
     const signal = this._uiManager.combinedSignal(ac);
 
     if (isSelected) {
-      this.div.classList.add("moving");
       this.#prevDragX = event.clientX;
       this.#prevDragY = event.clientY;
-      const pointerMoveCallback = e => {
-        const { clientX: x, clientY: y } = e;
-        const [tx, ty] = this.screenToPageTranslation(
-          x - this.#prevDragX,
-          y - this.#prevDragY
-        );
-        this.#prevDragX = x;
-        this.#prevDragY = y;
-        this._uiManager.dragSelectedEditors(tx, ty);
-      };
-      window.addEventListener("pointermove", pointerMoveCallback, {
-        passive: true,
-        capture: true,
-        signal,
-      });
+      this.#dragPointerId = event.pointerId;
+      window.addEventListener(
+        "pointermove",
+        e => {
+          const { clientX: x, clientY: y, pointerId } = e;
+          if (pointerId !== this.#dragPointerId) {
+            e.preventDefault();
+            e.stopPropagation();
```

Comment:
This pattern seems to be repeated a bunch of times below, how about introducing a small helper function and use that everywhere instead?

E.g.
```javascript
const stopEvent = e => {
  e.preventDefault();
  e.stopPropagation();
};
```
and we can change this code to
```suggestion
            stopEvent(e);
```